src/util/expr.h

```diff
@@ -319,6 +319,40 @@ class exprt:public irept
   const_unique_depth_iteratort unique_depth_cend() const;
 };
 
+/// Protected class for all expressions.
```

Comment:
What is a "protected" class? When should anyone use it? I think this comment needs to be reworded.

Reply:
I'm a bit confused, an intermittent version had more elaborate documentation. Do a `git reflog` to look at those?